build.sbt

```diff
@@ -509,7 +509,10 @@ lazy val buildTools = project
       Map(
         "SCIP_JAVA_CLI" -> ((cli / pack).value / "bin" / "scip-java").toString
       ),
-    Test / fork := true
+    Test / fork := true,
+    // Our CI set up is a couple of measly vCPUs so parallelising tests there makes
```

Comment:
We can probably get access to more powerful buildkite machines if we want to. I never looked into it.